src/runtime_src/core/edge/drm/zocl/zocl_drv.c

```diff
@@ -814,6 +814,56 @@ static int zocl_iommu_init(struct drm_zocl_dev *zdev,
 	return 0;
 }
 
+static int zocl_gem_prime_mmap(struct drm_gem_object *obj,
+		struct vm_area_struct *vma)
+{
+	struct drm_zocl_bo *zocl_obj;
```

Comment:
Make it oneline like other place.
i.e. struct drm_zocl_bo *zocl_obj = to_zocl_bo(obj);